### PR TITLE
OPSEXP-2066 refactor amps updates in updatecli

### PR DESCRIPTION
**Pull request by gionn**
Ref: OPSEXP-2066

### CODE REVIEW

File: docs/developer-guide.md [hunk 1]
```diff
@@ -134,18 +134,25 @@ The OS support table in `docs/README.md` must be updates and so does the
...
 are multiple values files depending on which acs major version we want to check
 for any new minor release.
 
 Run updatecli with:
 
 ```bash
-updatecli apply --config scripts/updatecli/updatecli_config.yml --values scripts/updatecli/updatecli_acs73.yml
+updatecli apply --config scripts/updatecli/updatecli_config.tpl --values scripts/updatecli/updatecli_acsXX.yml
+updatecli apply --config scripts/updatecli/updatecli_config.tpl --values scripts/updatecli/updatecli_acsXX.yml --values scripts/updatecli/updatecli_tests_amps_target.yml
 ```
```
Comment (gionn) [inline docs/developer-guide.md:150]: I still need to handle how to skip ags when updating the main group_vars, suggestions welcome

File: scripts/updatecli/updatecli_config.tpl [hunk 2]
```diff
@@ -81,6 +86,16 @@ sources:
       versionFilter:
         kind: regex
         pattern: '{{ .adw.version }}(.(\d+))+{{ .version_pattern }}'
+  agsAmp:
+    name: AGS AMP {{ .ags.version }}.x
+    kind: gittag
+    scmid: acsEntRepo
```
Comment (alxgomz) [inline scripts/updatecli/updatecli_config.tpl:92]: https://github.com/Alfresco/governance-services seem to have more recent/consostent tags

File: scripts/updatecli/updatecli_config.tpl [hunk 3]
```diff
@@ -7,6 +7,12 @@ scms:
         url: https://github.com/Alfresco/acs-packaging.git
         branch: master
         directory: '/tmp/updatecli/acs'
+  acsEntRepo:
+    kind: git
+    spec:
+        url: git@github.com:Alfresco/alfresco-enterprise-repo.git
```
Comment (alxgomz) [inline scripts/updatecli/updatecli_config.tpl:13]:
```suggestion
        url: git@github.com/Alfresco/alfresco-enterprise-repo.git
```

File: docs/developer-guide.md (hunk 1 above)
Comment (alxgomz) [reply docs/developer-guide.md:150]: you could turn `.target.amps` from a bool to an array of some kind that contains the list of amps present in the `target_file` and then `{{ range }}` on that array in the template to render all the required target (which are all the same expect from the AMP name if I saw correctly)

**Review by alxgomz — CHANGES_REQUESTED** [review]

**Review by slohe1 — APPROVED** [review]

**Pull request merged**